monkestation/code/game/objects/effects/spawners/roomspawner.dm

```diff
@@ -12,7 +12,9 @@
 	dedicated_rooms = list(/datum/map_template/random_room/box_default_bar = 1,
 							/datum/map_template/random_room/box_clown_bar = 1,
 							/datum/map_template/random_room/box_syndi_bar = 1,
-							/datum/map_template/random_room/box_vietmoth_bar = 1)
+							/datum/map_template/random_room/box_vietmoth_bar = 1,
+							/datum/map_template/random_room/box_bloody_bar = 1,
+							/datum/map_template/random_room/box_clockwork_bar = 1)
```

Comment:
Remove the leftmost table and chair, pushing the middle table left a bit.
Will give more walkway.

Reply:
![Screenshot (177)](https://user-images.githubusercontent.com/69217972/168537086-eb37e737-5dec-4331-a43d-f992ae3e50be.png)